Fill naming jobs best practices documentation (#149)
yokai-php/batch-src: docs/docs/best-practices/naming-jobs.rst

Decide on a convention
------------------------------------------------------------

There is no specific recommandation, you can use whichever notation you prefer, such as:

* ``ImportUser``
* ``import_user``
* ``import.user``
* or any other format

However, we strongly recommend that you choose a convention and adhere to it consistently.

In a Symfony project
------------------------------------------------------------

> | As required by the Framework works, ``Jobs`` must be registered as services.

Reviewer: I don't agree on this one

Reviewer: ```suggestion
| Because of the way the framework works, ``Jobs`` must be registered as services.
```
